BaseFilter/sections/specific.txt

```diff
@@ -343,6 +343,8 @@ $xmlhttprequest,third-party,denyallow=rsc.cdn77.org|linkvertise.com|fastly.net|s
 !
 ! NOTE: Specific rules
 !
+celebfapper.com##.row
+/invoke.js$script,domain=celebfapper.com
```

Comment:
Only useful for iOS due to missing regexp support ( existing rule ` /[0-9a-f]{32}\/invoke\.js/$script,third-party,redirect-rule=noopjs`)

Reply:
Remove this too, base filter blocked it already too